lib/ecto/schema.ex

```diff
@@ -2002,6 +1979,32 @@ defmodule Ecto.Schema do
     end
   end
 
+  @doc false
+  def __define_timestamps__(mod, timestamps) do
```

Comment:
Couldn't name it `__timestamps__` (like `__field__` or `__has_one__`) because we already have a `__timestamps__` function in this module, used for timestamp generation.